### PR TITLE
Add missing variables to schedule solvers

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/thoth-application/issues/127

Fixes: https://github.com/thoth-station/investigator/issues/50

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Add missing env variables to schedule solvers

### CODE REVIEW

File: investigator/base/deploymentconfig.yaml [hunk 1]
```diff
@@ -59,6 +59,40 @@ spec:
...
+                  name: ceph
+            - name: THOTH_DEPLOYMENT_NAME
+              valueFrom:
+                configMapKeyRef:
+                  key: deployment-name
+                  name: thoth
+            - name: KUBERNETES_API_URL
+              value: "https://kubernetes.default.svc.cluster.local"
+            - name: KUBERNETES_VERIFY_TLS
+              value: "0"
```
Comment (fridex) [inline investigator/base/deploymentconfig.yaml:95]: Is this in cluster issue?

Comment (pacospace) [reply investigator/base/deploymentconfig.yaml:95]: > Is this in cluster issue?

I saw this error message: 

```
/template.openshift.io/v1/namespaces/thoth-test-core/processedtemplates (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:897)'),))`
```

Comment (fridex) [reply investigator/base/deploymentconfig.yaml:95]: Sounds like certification issue in the cluster.

Comment (harshad16) [reply investigator/base/deploymentconfig.yaml:95]: @fridex , any suggestion on how to resolve this in cluster ?
or should we go ahead with this solution of disabling it?

Comment (fridex) [reply investigator/base/deploymentconfig.yaml:95]: Let's disable it for now.

Comment (harshad16) [reply investigator/base/deploymentconfig.yaml:95]: thanks for looking into this @fridex

**Pull request merged**